.github/workflows/industrial_ci_action.yaml

```
@@ -0,0 +1,68 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+      - ros2
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - ROS_DISTRO: melodic
+            ROS_REPO: main
+          - ROS_DISTRO: melodic
+            ROS_REPO: testing
+            CATKIN_LINT: true
```

Let's add Noetic to the matrix as well:
```suggestion
          - ROS_DISTRO: noetic
            ROS_REPO: testing
            CATKIN_LINT: true
          - ROS_DISTRO: melodic
            ROS_REPO: main
```